src/protos/io/defang/v1/fabric.proto

```diff
@@ -105,9 +105,16 @@ message TrackRequest {
   string arch = 5;
 }
 
+enum Behavior {
+  DEVELOPMENT = 0;
```

Comment:
My bad: we should stick to `UNSPECIFIED = 0` for all enums

Reply:
Addressed in https://github.com/DefangLabs/defang/pull/627/files